Jon-Jonrosario5/Week-2/superMarket_Website/Quik Stop Market_files/main.js

```diff
@@ -0,0 +1,41 @@
+// Drop Down Section
+
+function showDropDown() {
+  document.getElementById("cereal").classList.toggle("show");
+}
+
+function showDropDown1() {
+  document.getElementById("candy").classList.toggle("show");
+}
+
+function showDropDown3() {
+  document.getElementById("beverage").classList.toggle("show");
+}
+
+function showMenu() {
+  document.getElementById("snack").classList.toggle("show");
+}
+
+function showCheckout(){
+  document.getElementById("checkOutItems").classList.toggle("show");
+}
+
+window.onclick = function(event) {
+  if (!event.target.matches('.dropbtn')) {
+    var dropdowns = document.getElementById("dropdown-content");
+    var i;
+    for (i = 0; i < dropdowns.length; i++) {
```

Comment:
It's okay to declare `i` within the iterator of your loop. 

Is there a cleaner way to go about this logic?